config/jobs/kubernetes/generated/generated.yaml

```diff
@@ -1,4 +1,231 @@
 periodics:
+- tags:
+  - generated   # AUTO-GENERATED by releng/generate_tests.py - DO NOT EDIT!
+  interval: 1h
+  labels:
+    preset-service-account: 'true'
+    preset-k8s-ssh: 'true'
+  name: ci-kubernetes-e2e-gce-cos-k8sbeta-alphafeatures
+  spec:
+    containers:
+    - args:
+      - --bare
+      - --timeout=200
+      - --scenario=kubernetes_e2e
+      - --
+      - --cluster=test-eef92b583b
+      - --check-leaked-resources
+      - --provider=gce
+      - --gcp-zone=us-west1-b
+      - --gcp-node-image=gci
+      - --extract=ci/latest-1.27
```

Comment:
Th is also supposed to be `latest-1.28` I think.

Reply:
Done.